AlfrescoApp/View Controllers/Profile Selection View Controller/ProfileSelectionViewController.m

```diff
@@ -205,6 +208,13 @@ - (void)loadData
                         if ([config.identifier isEqualToString:profile.identifier])
                         {
                             [self tableView:self.tableView didSelectRowAtIndexPath:[NSIndexPath indexPathForRow:i inSection:0]];
+                            
+                            if (isEmbeddedConfigurationLoaded)
```

Comment:
I don't think this is the correct check to be made here. If the a configuration file exists on the server (thus the bool isEmbeddedConfigurationLoaded would be NO) and the admin adds one more view to the default profile, the menu would not be reloaded. Perhaps a different check should be made or the menu should be reloaded for every case.